### PR TITLE
Added Events to front page

### PR DESCRIPTION
**Pull request by jd-campbell**
I added the following conferences/meetings

- CROPS 2024
- NAPB 2024
- PAG 2025

### CODE REVIEW

File: events/_posts/2024-06-03-CROPS.md
Comment (StevenCannon-USDA) [inline events/_posts/2024-06-03-CROPS.md:1]: Minor change: "CROP Conference 2024" --> "CROPS Conference 2024"

**Review by StevenCannon-USDA — COMMENTED** [review]
Looks good to me, but see one typo/minor error in the link to the CROPS conference

**Review by maxglycine — CHANGES_REQUESTED** [review]
It looks great, just 2 of the three links give an error in FF "
Firefox Can’t Open This Page

To protect your security, www.hudsonalpha.org will not allow Firefox to display the page if another site has embedded it. To see this page, you need to open it in a new window."

Maybe make the link open in a separate window using target="_blank"?

The PAG link opens a new window and that's fine with me.

Checked behavior in Chrome and it acts the same.  Don't know why it likes the P
AG link, but they don't like the Hudson and NAPB links

I was looking at the code and I could not find a layout = "event" in the _layout folder.  Could this be the problem?

**Comment by jd-campbell** [comment]
@maxglycine -- the  index.html file for events is found under the event folder.

**Pull request closed without merging**